### DISCUSSION TITLE
Updates on empty Servers

### DISCUSSION THREAD

**Discussion by Marius9308** [gameserverapp/Platform#457] [Suggestions]
would it be possible for servers to update automatically as soon as there are no more players on them? No new player can join until the update and a countdown on an empty server is not necessary. Best regards

**Comment by mvaessen** [comment]
hi, yes that is definitely an option we can add. Do note that in case people are just connecting to the server, they might get kicked. 

In the cases when there are RCON or Query port issues, this may prevent GSA from detecting players. That would immediately kick everyone from the server to perform the update.

**Reply by Marius9308** [reply]
There is always some risk. But I think just having the option is very good. Thanks very much!
